### PR TITLE
Creating of choose president 

### PR DESCRIPTION
**Pull request by CarlosBazan28**
Se agrego las tarjetas de los presidentes.

### CODE REVIEW

File: src/app/components/choose-president/choose-president.component.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    NzListModule,
+  ],
+  templateUrl: './choose-president.component.html',
+  styleUrl: './choose-president.component.scss',
+})
+export class ChoosePresidentComponent {
+  onBack(): void {
+    console.log('onBack')
+  }
+  presidents = [
```
Comment (jcastaneda06) [inline src/app/components/choose-president/choose-president.component.ts:26]: Add correct typing for president:
https://github.com/CorrupcionPA/CorrupcionPA-corrupcionpa-api/issues/1

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -1,13 +1,7 @@
 import { Routes } from '@angular/router'
 import { HomeComponent } from './components/home/home.component'
-import { ViewPresidentComponent } from './components/view-president/view-president.component'
```
Comment (jcastaneda06) [inline src/app/app.routes.ts:3]: Don't delete new changes

File: src/app/app.routes.ts [hunk 3]
```diff
@@ -1,13 +1,7 @@
...
-
+import { ChoosePresidentComponent } from './components/choose-president/choose-president.component'
 export const routes: Routes = [
-  { path: 'home', component: HomeComponent, data: { breadcrumb: 'Home' } },
-  { path: '', redirectTo: '/home', pathMatch: 'full' },
-  {
```
Comment (jcastaneda06) [inline src/app/app.routes.ts:8]: Don't delete new changes

**Review by jcastaneda06 — CHANGES_REQUESTED** [review]
Generally ok. But deleted most recent changes from `app.routes.ts`

File: src/app/components/choose-president/choose-president.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (CarlosBazan28) [reply src/app/components/choose-president/choose-president.component.ts:26]: Listo.

File: src/app/components/choose-president/choose-president.component.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    NzBreadCrumbModule,
+    NzPageHeaderModule,
+    NzListModule,
+  ],
+  templateUrl: './choose-president.component.html',
+  styleUrl: './choose-president.component.scss',
+})
+export class ChoosePresidentComponent {
+  onBack(): void {
+    console.log('onBack')
```
[the remark this answers is not in the document]
Comment (CarlosBazan28) [reply src/app/components/choose-president/choose-president.component.ts:24]: Hecho

File: src/app/components/choose-president/choose-president.component.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+h2{
+  font-size: 16px;
```
[the remark this answers is not in the document]
Comment (CarlosBazan28) [reply src/app/components/choose-president/choose-president.component.scss:2]: listonguis

File: src/app/components/choose-president/choose-president.component.scss [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+  text-align: center;
+  font-weight: bold;
+  margin-top: 20px;
```
[the remark this answers is not in the document]
Comment (CarlosBazan28) [reply src/app/components/choose-president/choose-president.component.scss:5]: listo

**Review by jcastaneda06 — APPROVED** [review]

**Pull request merged**